src/tests/test_api.py

```diff
@@ -59,25 +63,36 @@
     "/origin/grid-mapfile",
 ]
 
+JSON_ENDPOINTS = [
+    "/api/resource_group_summary",
+    '/miscproject/json',
+    '/miscsite/json',
+    '/miscfacility/json',
+    '/miscresource/json',
+    '/vosummary/json',
+    '/resources/stashcache-files',
+    '/stashcache/namespaces'
+]
+
 
 @pytest.fixture
-def client():
+def client() -> Generator[FlaskClient]:
```

Comment:
```suggestion
def client() -> Generator[FlaskClient, None, None]:
```
Sorry, apparently the syntax for generators is
```
Generator[yield type, send type, return type]
```